x-pack/plugins/maps/public/index.js

```diff
@@ -57,6 +58,7 @@ routes
     },
     resolve: {
       hasMaps: function (kbnUrl) {
+        addHelpMenuToAppChrome(chrome);
```

Comment:
It seems inconsistent that help menu is added from within map_controller but for the listing page it is added in the routing. Not sure which one I like better but lets pick one and be consistent across the application.

Maybe move this to https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/shared/components/map_listing.js to be consistent with map_controller